server/package.json

```diff
@@ -54,6 +54,7 @@
     "@types/jsonwebtoken": "~7.1.33",
     "@types/mocha": "~2.2.33",
     "@types/multer": "0.0.33",
+    "@types/node": "~8.0.12",
```

Comment:
i have found the solution [source](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/16261). process.env is only with string literals instead of dottes accessible since typescript 2.2.